src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/recallnotification/RecallNotificationContextFactory.kt

```diff
@@ -47,6 +47,7 @@ class RecallNotificationContextFactory(
 ) {
   fun createContext(recallId: RecallId, currentUserId: UserId): RecallNotificationContext {
     val recall = recallRepository.getByRecallId(recallId)
+    // FIXME - no longer need to look these values up
```

Comment:
I'd always prefer a `// FIXME` to reference a ticket ID but ... OK.

Reply:
Yep, it references THIS ticket :) .I have the checklist on the ticket itself.